src/screens/About/TeamTable/TeamTable.style.tsx

```diff
@@ -1,20 +1,33 @@
 import styled from 'styled-components';
 import Typography from '@material-ui/core/Typography';
 
-export const TeamTableGrid = styled.div``;
-export const TeamMember = styled.div`
+export const TeamTableGrid = styled.div<{
+  isTeam?: Boolean;
```

Comment:
Is `isTeam` used?  If not, please remove from the TeamTable.tsx code that passes them in too?